src/globus_cli/commands/streams/tunnel/update.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+import uuid
+
+import click
+import globus_sdk
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import OMITTABLE_STRING, command
+from globus_cli.termio import display
+
+from ._common import TUNNEL_STANDARD_FIELDS
+
+
+@command("create", short_help="Create a Globus Tunnel.")
+@click.argument("tunnel_id", metavar="TUNNEL_ID", type=click.UUID)
+@click.option(
+    "--label",
+    type=OMITTABLE_STRING,
+    default=globus_sdk.MISSING,
+    help="A friendly label to associate with the Tunnel.",
```

Comment:
Tunnel (as a name for the resource) appears in uppercase and lowercase in this PR. Recommending lowercase for consistency.

```suggestion
    help="A friendly label to associate with the tunnel.",
```